action.js

```diff
@@ -22,6 +22,8 @@ module.exports = async ({ context, core, exec, github }) => {
     pr_number = pr.number;
   } else if (context.eventName === "merge_group") {
     pr_number = parseInt(context.ref.split("/pr-")[1]);
+  } else {
+    pr_number = context.issue.number;
```

Comment:
This is a great call-out, thank you!

An `else {}` here would be ideal to capture any cases I didn't think of. Though, unfortunately, we can't reliably rely on `context.issue.number` alone since it's not available in all event payloads, such as `push` and `merge_group` event triggers.

Let's see if we can make this `else {}` a tad more robust.